test/support/analyzer_verification/check_source.ex

```diff
@@ -61,4 +61,36 @@ defmodule ElixirAnalyzer.Support.AnalyzerVerification.CheckSource do
       String.trim(code_string) == Code.format_string!(code_string) |> Enum.join()
     end
   end
+
+  check_source "should have an even number of bananas in the code" do
+    type :actionable
+    comment "even banana count"
+
+    check(%Source{code_string: code_string}) do
+      chunks = String.split(code_string, "banana", trim: false)
+      banana_count = Enum.count(chunks) - 1
+
+      if rem(banana_count, 2) == 1 do
+        {false, %{banana_count: banana_count}}
+      else
+        {true, %{banana_count: banana_count}}
+      end
+    end
+  end
+
+  check_source "should have an even number of bananas in the code" do
```

Comment:
Is this the same as above?

Reply:
lol, yes! I wonder what I was trying to write there

[`ed10546` (#374)](https://github.com/exercism/elixir-analyzer/pull/374/commits/ed1054606e2f7a2b98c518b37b661d36f110ef64)